src/libraries/System.Drawing.Common/src/System/Drawing/Printing/PrintingServices.Unix.cs

```diff
@@ -664,11 +664,11 @@ internal static void GetPrintDialogInfo(string printer, ref string port, ref str
 
                 NameValueCollection options = LoadPrinterOptions(cups_dests.options, cups_dests.num_options);
 
-                if (options["printer-state"] != null)
-                    state = int.Parse(options["printer-state"]!); // TODO-NULLABLE dotnet/roslyn#34644
+                if (options["printer-state"] is string printerState)
+                    state = int.Parse(printerState);
 
-                if (options["printer-comment"] != null)
-                    comment = options["printer-state"]!; // TODO-NULLABLE dotnet/roslyn#34644
+                if (options["printer-comment"] is string printerComment)
+                    comment = printerComment;
```

Comment:
It was previously `options["printer-state"]`, this looks like a bug in the old code, just want to make sure this is intentional

Reply:
Interesting, good catch, I hadn't noticed that.  This has to be a bug, though can't be that important, as it's been this way since the code was originally added to mono in 2007.

cc: @safern @tannergooding

Reply:
We haven't gotten a report from this, but it does make sense to take this fix. It is weird to include the printer state as the comment.

adding: @akoeplinger who might have more context

Reply:
Yeah I agree, it was probably never noticed since it's just a comment.